### ISSUE TITLE
Game crash loading new updated 4.2.4

### ISSUE THREAD

**Issue by Damalas75** [DarkPacks/SkyFactory-4#2020]
## Steps to Reproduce (for bugs)

1. Install the newest version 4.2.4
2. Load up in curse forge app
3. get this 
![image](https://user-images.githubusercontent.com/79496641/147492496-1b345112-c5ac-4bf4-a80e-9d39bc8daab0.png)

## Logs

* Client/Server Log:
[latest.log](https://github.com/DarkPacks/SkyFactory-4/files/7781082/latest.log)

* Crash Log:
[crash-2021-12-27_12.55.15-client.txt](https://github.com/DarkPacks/SkyFactory-4/files/7781052/crash-2021-12-27_12.55.15-client.txt)

## World Information ** N/A unable to load into world generation
* World Preset:
* Prestige Enabled:
* Modpack Version world created in:
* Additional Content Installed:

## Client Information
* Modpack Version:
* Java Version:
* Launcher Used:
* Memory Allocated:
* Server/LAN/Single Player:
* Optifine Installed:
* Shaders Enabled:

End Solution- Unable to play 4.2.4 version and must play an older version. (4.2.2)

**Comment by Damalas75** [comment]
also note I did an uninstall and a reinstall on the pack and both have have the same result for the 4.2.4 Version

**Comment by sam-kirby** [comment]
It looks like this might be down to your download of EnderCore being corrupt; can you try downloading that mod manually and substituting it?

**Comment by Damalas75** [comment]
yep that fixed the issue, guessing something was going wrong with the core download on the curse forge app so did it manually in there.
